### ISSUE TITLE
Unable to start server: Input error: Device or resource busy (os error 16)

### ISSUE THREAD

**Issue by k1llerk3ks** [htrefil/rkvm#36]
Hey, Barrier Refugee on wayland here. I wanna run the server on my desktop PC (device A) and connect my Laptop as client (device B). 

I've used the AUR installation and installed the `rkvm` package. Running Manjaro Linux on both devices.

After creating certificate and Key and running `sudo cp /etc/rkvm/server.example.toml /etc/rkvm/server.toml`,

 i can run the server start command by typing `sudo rkvm-server /etc/rkvm/server.toml`. After a few seconds the server stops, because of an Input Error. See the log:
 
 ```
sudo rkvm-server /etc/rkvm/server.toml                    
[sudo] Password for k1llerk3ks: 
[INFO  rkvm_server::server] Listening on 192.168.133.33:5258
[INFO  rkvm_input::event_reader] Registered helper for /dev/input/event10 at /dev/input/event4
[INFO  rkvm_input::event_reader] Registered helper for /dev/input/event9 at /dev/input/event27
[INFO  rkvm_input::event_reader] Registered helper for /dev/input/event8 at /dev/input/event28
[INFO  rkvm_input::event_reader] Registered helper for /dev/input/event7 at /dev/input/event29
[INFO  rkvm_input::event_reader] Registered helper for /dev/input/event6 at /dev/input/event30
[ERROR rkvm_server] Error: Input error: Device or resource busy (os error 16)
```

For testing purposes i tried running the server on the Laptop and using the Desktop PC as Client. This works well.  The Desktop PC is running wayland, the Laptop is still on X11.

I am thankful for any assistance and help!

EDIT: Same outcome, if i use X11 on the Server/Desktop PC.

**Comment by htrefil** [comment]
Thanks for reporting this, I'll take a look.

**Comment by htrefil** [comment]
Looks like this got fixed in 0.5.0.
Can you please confirm this is the case by running the latest rkvm?

**Comment by k1llerk3ks** [comment]
@htrefil I am now using `rkvm-0.6.0-1` from the Arch AUR. I've tested with X11 on both PCs and now with Wayland on the serverside and X11 on Client. This really works wonderfully nice now. Only thing i am missing now is a shared Clipboard, but that's not part of this issue. Thanks for your Update! Now i am ready to use Wayland! :partying_face:  I'll close this Issue now.
